pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -30,6 +30,11 @@
       <TargetFramework>.NETCore50</TargetFramework>
     </Dependency>
 
+    <Dependency Include="Microsoft.Net.UWPCoreRuntimeSdk">
```

Comment:
No. https://github.com/dotnet/core-setup/blob/release/1.1.0/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj#L11 is the way to do it.